<launch>
	<arg name="agent_num" default="6" />
	
	<node pkg="sunray_swarm" name="swarm_formation_ugv" type="swarm_formation" output="screen">
		<param name="agent_type" value="1" />
		<param name="agent_num" value="$(arg agent_num)" />
		<param name="formation_keep_time" value="5.0" />
	</node>
</launch>